Corrections.py

```diff
@@ -1,13 +1,17 @@
 import os
 import ROOT
 import yaml
+import itertools
 from RunKit.sh_tools import sh_call
+from Common.Utilities import *
```

Comment:
why do we need `Common.Utilities` here?

Reply:
bc in utilities the ```generate_enum_class``` is defined

Reply:
ok I applied the following comment so no more needed to import utilities